app/css/authentication.css

```diff
@@ -0,0 +1,26 @@
+.AuthForm {
+    padding: 15px;
+}
+.AuthForm > p {
```

Comment:
Can you make that `.AuthForm-description` and set the appropriate classes where needed.